sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import.service.ts

```diff
@@ -0,0 +1,108 @@
+import { Injectable } from "@nestjs/common";
+import { ApplicationWithdrawalTextValidationResult } from "./application-bulk-withdrawal-import-text.models";
+import { In, Repository } from "typeorm";
+import { Application } from "@sims/sims-db";
+import {
+  ApplicationBulkWithdrawalValidationModel,
+  ApplicationData,
+} from "./application-bulk-withdrawal-validation.models";
+import { InjectRepository } from "@nestjs/typeorm";
+
+type ApplicationDataMap = Record<string, ApplicationData>;
+
+/**
+ * Handles the application withdrawal bulk insert preparation.
+ */
+@Injectable()
+export class ApplicationBulkWithdrawalImportService {
+  constructor(
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>,
+  ) {}
+
+  /**
+   * Generates the application bulk withdrawal validation model.
+   * @param textValidations text validation models to be converted to the validation models.
+   * @param originator institution that sent the file.
+   * @param institutionId institution id.
+   * @returns application bulk withdrawal models to be validated and persisted.
+   */
+  async generateApplicationBulkWithdrawalValidationModels(
```

Comment:
Not a blocker but `generateApplicationBulkWithdrawalValidationModels` inside a service already named `ApplicationBulkWithdrawalImport` can be renamed to `generateValidationModels`.